.golangci.yaml.in

```
@@ -0,0 +1,204 @@
+        - pkg: github.com/gardener/gardener/extensions/pkg/controller/([^m]\w+)
+          alias: extensions${1}controller
+        - pkg: github.com/gardener/gardener/extensions/pkg/controller/heartbeat/cmd
+          alias: extensionsheartbeatcmd
+        - pkg: github.com/gardener/gardener/extensions/pkg/util/secret/manager
+          alias: extensionssecretsmanager
+        - pkg: github.com/gardener/gardener/extensions/pkg/webhook
+          alias: extensionswebhook
+        - pkg: github.com/gardener/gardener/extensions/pkg/webhook/([^m]\w+)
+          alias: extensions${1}webhook
+        # Gardener other packages
+        - pkg: github.com/gardener/gardener/pkg/apis/core
+          alias: gardencore
+        - pkg: github.com/gardener/gardener/pkg/apis/core/([\w\d]+)
+          alias: gardencore${1}
+        - pkg: github.com/gardener/gardener/pkg/apis/core/([\w\d]+)/helper
+          alias: ${1}helper
+        - pkg: github.com/gardener/gardener/pkg/apis/core/([\w\d]+)/constants
+          alias: ${1}constants
+        - pkg: github.com/gardener/gardener/pkg/apis/([^c]\w+)/([\w\d]+)
+          alias: $1$2
+        - pkg: github.com/gardener/gardener/pkg/apis/([^c]\w+)/([\w\d]+)/([\w\d]+)
+          alias: $1$2$3
+        - pkg: github.com/gardener/gardener/pkg/\w+/controller/([\w\d]+)
+          alias: ${1}controller
+        - pkg: github.com/gardener/gardener/pkg/\w+/webhook/([\w\d]+)
+          alias: ${1}webhook
+        - pkg: github.com/gardener/gardener/pkg/chartrenderer
+          alias: chartrenderer
+        - pkg: github.com/gardener/gardener/pkg/client/kubernetes
+          alias: kubernetesclient
+        - pkg: github.com/gardener/gardener/pkg/extensions
+          alias: gardenerextensions
+        - pkg: github.com/gardener/gardener/pkg/healthz
+          alias: gardenerhealthz
+        - pkg: github.com/gardener/gardener/third_party/mock/\w+/(\w+)
+          alias: mock${1}
+        - pkg: github.com/gardener/gardener/pkg/component/(\w+)/constants
+          alias: ${1}constants
+        - pkg: github.com/gardener/gardener/pkg/utils/(\w+)
+          alias: ${1}utils
+        - pkg: github.com/gardener/gardener/pkg/utils/secrets/manager
+          alias: secretsmanager
+        - pkg: github.com/gardener/gardener/pkg/utils/secrets/manager/fake
+          alias: fakesecretsmanager
+        # Registry cache packages
+        - pkg: github.com/gardener/gardener-extension-registry-cache/pkg/apis/registry
+          alias: registryapi
+        - pkg: github.com/gardener/gardener-extension-registry-cache/pkg/apis/registry/([\w\d]+)
+          alias: registry${1}
+        - pkg: github.com/gardener/gardener-extension-registry-cache/pkg/apis/mirror
+          alias: mirrorapi
+        - pkg: github.com/gardener/gardener-extension-registry-cache/pkg/apis/mirror/([\w\d]+)
+          alias: mirror${1}
+        - pkg: github.com/gardener/gardener-extension-registry-cache/pkg/apis/([^c]\w+)/([\w\d]+)
```

What is the reason for the `[^c]` part?

I guess, it exists for `github.com/gardener/gardener/pkg/apis/([^c]\w+)/([\w\d]+)` in order to exclude `github.com/gardener/gardener/pkg/apis/core`.

For our repo, I don't see a concrete reason why we should have this special handlig:

```suggestion
        - pkg: github.com/gardener/gardener-extension-registry-cache/pkg/apis/(\w+)/([\w\d]+)
```